src/topology/category/UniformSpace.lean

```diff
@@ -89,6 +93,7 @@ induced_category.has_forget₂ to_UniformSpace
 end CpltSepUniformSpace
 
 namespace UniformSpace
+local notation `forget` := (forget₂ CpltSepUniformSpace UniformSpace).obj
```

Comment:
This notation is a bit confusing because there is already something else called `forget` which makes sense in this context. If you just want some short notation, how about `U`?

Reply:
Sounds good.